problems/sevenBoom.js

```diff
@@ -4,8 +4,18 @@
 * @param {number} n - The number to count up to
 * @returns {number[]} - An array matching the pattern described above
 */
-function sevenBoom() {
-
+function sevenBoom(n) {
+  let arr = []
+  for (let i = 1; i <= n; i++) {
+      if (i % 7 === 0) {
+          arr.push("BOOM")
+      } else if (i % 10 === 7) {
```

Comment:
Explore finding another conditional that would work if n = 80. Would this conditional give you "BOOM" for 70-79?